symfony/src/Service/SiteService.php

```diff
@@ -2,38 +2,86 @@
 
 namespace App\Service;
 
+use App\Entity\Awardee;
+use App\Entity\Organization;
 use App\Entity\Site;
 use Doctrine\ORM\EntityManagerInterface;
 use Symfony\Component\DependencyInjection\ParameterBag\ParameterBagInterface;
 use Symfony\Component\HttpFoundation\Session\SessionInterface;
 
 class SiteService
 {
+    /**
+     * @var ParameterBagInterface
+     */
     private $params;
+    /**
+     * @var SessionInterface
+     */
     private $session;
+    /**
+     * @var EntityManagerInterface
+     */
     private $em;
+    /**
+     * @var array
+     */
+    protected $siteNameMapper = [];
+    /**
+     * @var array
+     */
+    protected $organizationNameMapper = [];
+    /**
+     * @var array
+     */
+    protected $awardeeNameMapper = [];
 
+    /**
+     * SiteService constructor.
+     * @param ParameterBagInterface $params
+     * @param SessionInterface $session
+     * @param EntityManagerInterface $em
+     */
     public function __construct(ParameterBagInterface $params, SessionInterface $session, EntityManagerInterface $em)
     {
         $this->params = $params;
         $this->session = $session;
         $this->em = $em;
     }
 
+    /**
+     * @return bool
+     */
     public function isTestSite(): bool
     {
         return $this->params->has('disable_test_access') && !empty($this->params->get('disable_test_access')) && $this->session->get('siteAwardeeId') === 'TEST';
     }
 
+    /**
+     * @return null
+     */
     public function getSiteId()
     {
         if ($site = $this->session->get('site')) {
             return $site->id;
-        } else {
-            return null;
         }
+        return null;
+    }
+
+    /**
+     * @return null
+     */
```

Comment:
I commented earlier about removing these in favor of return types, but wanted to also note that these two are incorrect.